### DISCUSSION TITLE
Custom Preset bug in Handbrake

### DISCUSSION THREAD

**Discussion by ghost** [community/community#45854] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello. I was redirected to this site when I clicked a link at the Handbrake site to report a possible bug in the program. I use Handbrake primarily to convert video formats. I recently installed Handbrake in my new MacBook Pro running Ventura. The Handbrake I am using is Version 1.6.1 (2023012200). The problem I am having is that when I create a custom preset it will not retain the audio settings. I do not know if this has been reported or not. I did not have this problem when I was using Mac High Sierra. If it is a bug please alert the Handbrake developers so that the problem can be corrected in the next update. Thank you.

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
